pyproject.toml

```diff
@@ -133,6 +131,8 @@ select = [
     "F",
     "W",
 ]
+# Disable autofix for unused imports (`F401`).
+unfixable = ["F401"]
```

Comment:
Why?

Reply:
If you have your editor set up to do them for you, it can be annoying to have it remove imports that are already written but not used yet.

I’m OK with both, I’ve changed my habits to never do the above, but I thought it might be annoying for others. Your call!

Reply:
I would really prefer that it makes the change on pre-commit